.travis.yml

```diff
@@ -6,7 +6,7 @@ go:
 - "1.13"
 script:
 - echo "Building ingress controller commit:${TRAVIS_COMMIT}"
-- make BUILD_IN_CONTAINER=0 container;
+- make container BUILD_IN_CONTAINER=0 DOCKER_BUILDKIT=0;
```

Comment:
why `DOCKER_BUILDKIT=0`?

Reply:
TravisCI comes with docker 18.06. So by default it fails due to docker version.

If you try to update docker you would inexplicably get: 
```bash
#3 [internal] load metadata for docker.io/library/nginx:1.17.7
#3 ERROR: docker.io/library/nginx:1.17.7 not found
#7 [internal] load build context
#7 transferring context: 7.59MB 0.1s done
#7 CANCELED
#4 [base 1/4] FROM docker.io/library/nginx:1.17.7
#4 resolve docker.io/library/nginx:1.17.7 0.1s done
#4 ERROR: docker.io/library/nginx:1.17.7 not found
```
(https://travis-ci.org/nginxinc/kubernetes-ingress/builds/639443840?utm_source=github_status&utm_medium=notification)

So BUILDKIT is disabled just for travis in order for the build to succeed.